exercises/practice/meetup/.meta/test_template.erb

```diff
@@ -0,0 +1,12 @@
+require 'minitest/autorun'
+require_relative 'meetup'
+
+class MeetupTest < Minitest::Test
+<% json["cases"].each do |cases| %>
+    def test_<%= underscore(cases["description"]) %>
+        <%= skip? %>
+        meetup = Meetup.new(<%= cases["input"]["month"] %>, <%= cases["input"]["year"] %>).day(:<%= cases["input"]["dayofweek"].downcase %>, :<%= cases["input"]["week"] %>)
```

Comment:
Conceptually, and as a pattern, the tests are `assert expected, actual` and we can use these names.  If we are careful with this, a high percentage of template portions will follow this pattern, the only difference being what is referenced by `actual` as to what is called, while the expected is always what we expect.